python/ray/experimental/sgd/pytorch/pytorch_trainer.py

```diff
@@ -11,12 +11,10 @@
 import ray
 
 from ray.tune import Trainable
-from ray.tune.resources import Resources
-from ray.experimental.sgd.pytorch.pytorch_runner import PyTorchRunner
-from ray.experimental.sgd.pytorch.distributed_pytorch_runner import (
-    DistributedPyTorchRunner)
-from ray.experimental.sgd.pytorch import pytorch_utils
-from ray.experimental.sgd import utils
```

Comment:
some things like this need to be kept

Reply:
fixed but might possibly be missing imports (probably not)